### ISSUE TITLE
Restructure conditionals

### ISSUE THREAD

**Issue by rreinold** [WorkForPizza/goserve#3]
if os.IsPermission(err) {
	sendResp(conn, reqLine, 403, nil)
	return
} else {
	sendResp(conn, reqLine, 500, nil)
	fmt.Println(err)
	return
}

=>

errorCode := DEFAULT_ERROR
if os.IsPermission(err) {
	errorCode = 403
}
sendResp(conn, reqLine, errorCode, nil)
return

**Comment by WorkForPizza** [comment]
36cbb5cf065e7bebc19b6f041babc869188be662

**Comment by WyattHRichter** [comment]
I agree with @rreinold, this was a pretty amateur mistake on your part @WorkForPizza. Next time you consider committing code, stop for a second and _think_.
